client/cli/src/main.rs

```diff
@@ -59,7 +59,7 @@ struct Args {
 }
 
 fn main() {
-    let _logger = set_default_global_logger(false /* async */, None);
+    let _logger = set_global_logger_default(false /* async */, None);
```

Comment:
maybe just set_global_logger ?

Reply:
I originally tried this but it conflicts with slog_scope::set_global_logger() so I wasn't sure if it made things clearer or not. In the end, I've changed this to "set_global_logger" and then made the slog calls explicit.